dev/main.go

```diff
@@ -0,0 +1,5 @@
+package main // import "storj.io/storj/dev"
+
+import (
+  "github.com/golang/protobuf/protoc-gen-go"
```

Comment:
I'm thinking the linter will complain, should we make it an anonymous import ?

Reply:
What's an anonymous import look like? Alternatively we could possibly exclude this directory from the linter.

Reply:
add an underscore. My IDE runs goimports on save so it will just remove the line everytime I press save otherwise

Reply:
```
To import a package solely for its side-effects (initialization), use the blank identifier as explicit package name:
```

I think that's the right way to do it anyways